Bottom: 7407ef96b2c4b466ccba0cce99eb7f6be143164b
Top:    99544d6def39ec45ab4a52fcde8092fda2eecf87
Author: Igor Stoppa <igor.stoppa@huawei.com>
Date:   2018-06-07 16:06:51 +0300

Use designated initializer for struct list_head in list.h


---

diff --git a/include/linux/list.h b/include/linux/list.h
index 4b129df4d46b..11847c96571f 100644
--- a/include/linux/list.h
+++ b/include/linux/list.h
@@ -18,7 +18,10 @@
  * using the generic single-entry routines.
  */
 
-#define LIST_HEAD_INIT(name) { &(name), &(name) }
+#define LIST_HEAD_INIT(name) {	\
+	.next = &(name),	\
+	.prev = &(name),	\
+}
 
 #define LIST_HEAD(name) \
 	struct list_head name = LIST_HEAD_INIT(name)
